rest/config.go

```diff
@@ -331,7 +334,73 @@ func (dbConfig *DbConfig) setup(name string) error {
 		dbConfig.Server = &urlStr
 	}
 
-	return err
+	// If the sync function is referenced from an external file or http(s) endpoint
+	// unlike the inline JavaScript definition, read it from the specified path.
+	if dbConfig.Sync != nil {
+		sync, err := loadJavaScript(*dbConfig.Sync)
+		if err != nil {
+			return err
```

Comment:
It doesn't look like we're including any context for the error.  So if one of multiple external javascript definitions fails with an HTTP error, can a user easily identify which one is the problem?

Reply:
Introduced a custom error to easily identify the underlying problem when multiple external javascript definitions fails with an HTTP error.